lib/create_response.js

```diff
@@ -37,7 +38,7 @@ function createResponse(message) {
 
   return new Response(responseBodyOrNull, {
     status: message.statusCode,
-    statusText: message.statusMessage,
+    statusText: STATUS_CODES[message.statusCode],
```

Comment:
why not?

```suggestion
    statusText: message.statusMessage || STATUS_CODES[message.statusCode],
```

Reply:
@Uzlopak I'm not against adding this, but from what I see, we never set this, so it will always be `null`

Reply:
I'd say if `STATUS_CODES[message.statusCode]` passes all our tests, I'd go with that.

Reply:
small note: This is fetch-only code, so we don't have a big coverage here.